### PR TITLE
update README to include treatment info

### PR DESCRIPTION
**Pull request by MikeLippincott**
Added a table of treatments for interpret ability of  analysis. Please let me know if this makes any sense?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,45 @@
...
+| H202              | 2                 | Cell Death                    |
+| Flagellin         | 3                 | Induce Pyroptosis             |
+| LPS               | 5                 | Induce Pyroptosis             |
+| LPS + Nigericin   | 2 * 3 = (6)       | Induce Pyroptosis             |
+| Disulfiram        | 3                 | Pyroptosis Inhibitor          |
+
+
+# Other Treatments and combinations
+
+| Treatments        | Number of Doses   | Inhibitors | Number of Doses | Function                      |
```
Comment (jenna-tomkinson) [inline README.md:26]: Now this took me a bit to understand.

For example, above, when you mention the `treatments of interest`, you state the doses of LPS+Nigercin are 6 in total, but then you now have 2 different LPS+Nigercin doses below this.

I am assuming this just means that out of the 6 total combination of doses, there is only one of those combinations with this inhibitor.

Does it make sense to mention what combination of doses this is specifically? Is it a high+high, low+high, etc.?

I can see this being confusing to someone who doesn't know this dataset as well.

Another example I could see is that above you have three doses total for Flagellin, but here you only have **two** doses used in combination with an inhibitor. Where did the other dose go?

**Review by jenna-tomkinson — COMMENTED** [review]
Overall, I really like the layout you have created to explain everything! The tables are a great idea!

I have added some comments regarding some of the namings and how the doses are explained. It is a challenge to try and describe all of the different combinations of doses, but I think that is the main thing I am getting most confused about. Especially with LPS for example, only 1 dose of LPS is treated with an inhibitor like Z-VAD-FMK, is it important to know if this is a high or low dose of LPS?

Feel free to reach out to discuss!

**Comment by MikeLippincott** [comment]
@jenna-tomkinson I agree with all of these comments. Thank you for your input! I will make some changes and request a review tomorrow.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,45 @@
 # pyroptosis_signature_data_analysis
 Analysis Repository for the Interstellar project. For data processing for this project please see [pyroptosis_signature](https://github.com/WayScience/pyroptosis_signature)
 
+### Workflow:
+
+Cells were incubated for 1h in the inhibitors, and then incubated for 6h in each inducer.
+For more information on plate layout and experimental design please see [pyroptosis_signature](https://github.com/WayScience/pyroptosis_signature).
+
+
+# Treatments of Interest
```
[the remark this answers is not in the document]
Comment (MikeLippincott) [reply README.md:10]: resolved

File: README.md [hunk 3]
```diff
@@ -1,4 +1,43 @@
 # pyroptosis_signature_data_analysis
 Analysis Repository for the Interstellar project. For data processing for this project please see [pyroptosis_signature](https://github.com/WayScience/pyroptosis_signature)
 
+### Workflow:
+
+Cells were incubated for 1h in the inhibitors, and then incubated for 6h in each inducer.
+For more information on plate layout and experimental design please see [pyroptosis_signature](https://github.com/WayScience/pyroptosis_signature).
+
+
+# Treatments of Interest (All in this table are with 0.025% inhibitor)
```
Comment (jenna-tomkinson) [inline README.md:10]: Can you clarify what inhibitor (e.g., DMSO, Media, etc.).

**Review by jenna-tomkinson — APPROVED** [review]
LGTM! Great job with the tables to make it clear what the treatments are!

I do have one last comment for you to address before merging. This is definitely a clear representation of the dataset 🎊

**Pull request merged**